devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusPlatformTask.java

```diff
@@ -36,9 +38,22 @@
         super(description);
     }
 
+    private boolean enableRegistryClient;
+
+    @Option(option = "enableRegistryClient", description = "Temporary option to enable Quarkus project management tasks to use the Quarkus Extension Registry Client API")
```

Comment:
Maybe we should allow to enable this flag only through system properties, so it won't break tooling when this option is removed?

Reply:
Yes, good idea

Reply:
Done. Now it should be enabled in the same way as in Maven `-DenableRegistryClient`